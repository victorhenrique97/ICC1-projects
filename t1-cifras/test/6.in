141
LOREM IPSUM DOLOR SIT AMET, CONSECTETUR ADIPISCING ELIT. SUSPENDISSE VIVERRA METUS SIT AMET ELIT PELLENTESQUE, VITAE PORTA LEO COMMODO. NUNC.
1
13
