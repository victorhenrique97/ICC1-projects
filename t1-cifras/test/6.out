YBERZ VCFHZ QBYBE FVG NZRG, PBAFRPGRGHE NQVCVFPVAT RYVG. FHFCRAQVFFR IVIREEN ZRGHF FVG NZRG RYVG CRYYRAGRFDHR, IVGNR CBEGN YRB PBZZBQB. AHAP.
